cli/tests/pcluster/schemas/test_cluster_schema.py

```diff
@@ -1144,3 +1145,45 @@ def test_one_api(install_base_toolkit, install_hpc_toolkit, failure):
             OneApiSchema().load(config_dict)
     else:
         OneApiSchema().load(config_dict)
+
+
+@pytest.mark.parametrize(
+    "config_dict, failure_message",
+    [
+        (
+            {
+                "LambdaFunctionsVpcConfig": {
+                    "SubnetIds": ["subnet-8e482ce8"],
+                    "SecurityGroupIds": ["sg-028d73ae220157d96"],
+                },
+                "DisableSudoAccessForDefaultUser": "True",
+            },
+            None,
+        ),
+        (
+            {
+                "LambdaFunctionsVpcConfig": {
+                    "SubnetIds": ["subnet-8e482ce8"],
+                    "SecurityGroupIds": ["sg-028d73ae220157d96"],
+                },
+                "DisableSudoAccessForDefaultUser": "False",
+            },
+            None,
+        ),
+        ({"LambdaFunctionsVpcConfig": {"SubnetIds": ["subnet-8e482ce8"]}}, "Missing data for required field"),
```

Comment:
*Best Practice*
What's the missing data here? SecurityGroups?
I suggest to define the test parameters with id to explain what scenario the test case is covering. Example [here](https://github.com/aws/aws-parallelcluster/blob/develop/cli/tests/pcluster/config/test_update_policy.py#L46)
In a simple unit test it may be considered overkilling to add ids to test cases because the case parameters are self-explanatory, but still it may be helpful.